doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2992,9 +2996,39 @@ Deletes an ``AuthenticatedUser`` whose ``id``  is passed. ::
 
     DELETE http://$SERVER/api/admin/authenticatedUsers/id/$id
     
-Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted.
+Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted. If a user cannot be deleted for this reason, you can choose to :ref:`disable a user<disable-a-user>`.
     
+.. _disable-a-user:
+
+Disable a User
+~~~~~~~~~~~~~~
+
+Disables an ``AuthenticatedUser`` whose ``identifier`` (without the ``@`` sign) is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/$identifier
     
+Disables an ``AuthenticatedUser`` whose ``id``  is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/id/$id
+    
+Note: A primary purpose of most Dataverse installations is to serve an archive. In the archival space, there are best practices around the tracking of data access and the tracking of modifications to data and metadata. In support of these key workflows, a simple mechanism to delete users that have performed edit or access actions in the system is not provided. Providing a Disable User endpoint for users who have taken certain actions in the system alongside a Delete User endpoint to remove users that haven't taken certain actions in the system is by design.
+
+This is an irreversible action. There is no option to undisable a user.
+
+Disabling a user with this endpoint will:
+
+- Disable the user's ability to log in to the Dataverse installation. A message will be shown, stating that the account has been disabled.
```

Comment:
Just a heads up to @TaniaSchlatter and @mheppler that @djbrooke and I added this "a message will be shown" line yesterday after our meeting. Again, here's how it looks right now:

<img width="1265" alt="Screen Shot 2021-02-16 at 3 06 18 PM (1)" src="https://user-images.githubusercontent.com/21006/108387023-3374c800-71db-11eb-8821-20cddb93a174.png">

Reply:
Thanks - I think this is fine for now, we can review and alter directly in the bundle during review if needed

Reply:
Do we have a documented workflow (e.g. screenshots) anywhere of the oauth log in/sign up process for new users currently, in 5.3? @TaniaSchlatter @pdurbin

Reply:
@mheppler good question - not that I know of

Reply:
What I can offer at the moment is some screenshots from testing with GitHub:

<img width="1120" alt="Screen Shot 2021-02-18 at 2 08 02 PM" src="https://user-images.githubusercontent.com/21006/108408206-c10fe200-71f2-11eb-8bed-fcf6b0869989.png">
<img width="973" alt="Screen Shot 2021-02-16 at 2 38 12 PM (1)" src="https://user-images.githubusercontent.com/21006/108408199-bf461e80-71f2-11eb-8a69-67aff37293b4.png">
<img width="1267" alt="Screen Shot 2021-02-16 at 2 38 40 PM (1)" src="https://user-images.githubusercontent.com/21006/108408200-bfdeb500-71f2-11eb-9fcc-8048d7921258.png">

Then, if you click "convert your account" instead of "Create Account", you see something like this:

<img width="1265" alt="Screen Shot 2021-02-16 at 4 35 49 PM (1)" src="https://user-images.githubusercontent.com/21006/108408420-06341400-71f3-11eb-8691-9fc3af11a5bd.png">